### ISSUE TITLE
what Contents#iterate() is doing?

### ISSUE THREAD

**Issue by yegor256** [jcabi/jcabi-github#863]
I don't understand what `Contents#iterate()` is doing? This functionality is not provided by Github and we don't have an integration test for the method. I think it's obsolete and should be deleted.

**Comment by dmarkov** [comment]
@darkled you're holding this task for 12 days already... -30 added to your rating, current score is: [+60](http://www.netbout.com/b/35091?open=rating)

**Comment by darkled** [comment]
@dmarkov this issue as mentioned is closely related to #903. From my point of view, nothing can be done here before that issue is closed. And I am not sure what is this score rating about?

**Comment by darkled** [comment]
@dmarkov please, assign this task to someone else

**Comment by dmarkov** [comment]
> @dmarkov please, assign this task to someone else

@darkled I will assign somebody else to this issue

**Comment by dmarkov** [comment]
@darkled you've been working with this task for too long. I'm sorry, but I'll assign someone else. Please stop working with it. See our [no obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle; -60 added to your rating, current score is: [-90](http://www.netbout.com/b/35091?open=rating)

**Comment by darkled** [comment]
@dmarkov @yegor256 I am not sure these -60 are correct, since I've rejected the task 3 days ago, aren't they?

**Comment by dmarkov** [comment]
@maurezen it's your task, go ahead

**Comment by maurezen** [comment]
@dmarkov @yegor256 shouldn't there also be my account label on this ticket?

**Comment by maurezen** [comment]
@yegor256 @dmarkov this depends on #968 and probably on #903 as well (are not these two bugs the same problem?)  . What do we do with that? Is puzzle submission sufficient in this case?

Also are we talking about `RtContents` here? `Contents` is an interface.

**Comment by maurezen** [comment]
@yegor256 any comments?

**Comment by yegor256** [comment]
@maurezen I think we should create an integration test for `RtContents` and take it from there. If it passes, good, close the ticket. If it doesn't pass, try to fix it. If you can't, create a puzzle and close the ticket.

**Comment by maurezen** [comment]
@yegor256 created a pull request here #995

**Comment by maurezen** [comment]
@yegor256 pull request  #995 merged to master, please close the issue.

**Comment by yegor256** [comment]
@maurezen thanks!

**Comment by maurezen** [comment]
@yegor256 thanks!

@dmarkov the issues is closed.

**Comment by dmarkov** [comment]
once `863-4e783bd3` puzzle is resolved (later, in another ticket), this ticket will be fully complete

**Comment by maurezen** [comment]
@yegor256 @dmarkov  I'm confused as to what's the policy in this case. I believe in jcabi/jcabi-manifests#16 the ticket was considered complete once closed, despite the puzzle introduced. What way should it be?

**Comment by yegor256** [comment]
@maurezen this task is done, the puzzle will be fixed in another task, later

**Comment by dmarkov** [comment]
@maurezen **30 mins** sent to your balance (ID `50904270`), many thanks!. +30 added to your rating, at the moment it is: [+915](http://www.netbout.com/b/35102?open=rating)

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#unknown]() is still not solved.
